src/ontology/cl-edit.owl

```diff
@@ -8791,10 +8792,17 @@ SubClassOf(Annotation(oboInOwl:is_inferred "true") obo:CL_0000625 obo:CL_0000791
 
 # Class: obo:CL_0000626 (olfactory granule cell)
 
-AnnotationAssertion(Annotation(oboInOwl:hasDbXref "GOC:mah") obo:IAO_0000115 obo:CL_0000626 "Granule cell that is part of the olfactory bulb.")
+AnnotationAssertion(Annotation(oboInOwl:hasDbXref "GOC:mah") Annotation(oboInOwl:hasDbXref "doi:10.1038/s41598-018-27692-8") obo:IAO_0000115 obo:CL_0000626 "A granule cell that has a soma located in an olfactory bulb granule cell layer. An olfactory granule cell is an interneuron that lacks an axon, makes reciprocal dendro-dendritic synapses with mitral cells and tufted cells and is involved in the fine spatio-temporal tuning of the responses of these principal olfactory bulb neurons to odors.")
+AnnotationAssertion(Annotation(oboInOwl:hasDbXref "doi:10.1038/s41598-018-27692-8") oboInOwl:hasExactSynonym obo:CL_0000626 "olfactory bulb granule cell")
+AnnotationAssertion(Annotation(oboInOwl:hasDbXref "doi:10.1038/s41598-018-27692-8") rdfs:comment obo:CL_0000626 "Granule cells are the most abundant neuronal population in the olfactory bulb and are continuously renewed throughout life.")
 AnnotationAssertion(rdfs:label obo:CL_0000626 "olfactory granule cell")
-EquivalentClasses(obo:CL_0000626 ObjectIntersectionOf(obo:CL_0000120 ObjectSomeValuesFrom(obo:RO_0002100 obo:UBERON_0002264)))
-SubClassOf(obo:CL_0000626 obo:CL_0000120)
+EquivalentClasses(obo:CL_0000626 ObjectIntersectionOf(obo:CL_0000120 ObjectSomeValuesFrom(obo:RO_0002100 obo:UBERON_0005378)))
+SubClassOf(obo:CL_0000626 obo:CL_0000099)
+SubClassOf(obo:CL_0000626 ObjectSomeValuesFrom(obo:RO_0002103 obo:CL_1001502))
+SubClassOf(obo:CL_0000626 ObjectSomeValuesFrom(obo:RO_0002103 obo:CL_1001503))
+SubClassOf(obo:CL_0000626 ObjectSomeValuesFrom(obo:RO_0002120 obo:CL_1001502))
```

Comment:
This axiom is duplicated

Reply:
Can you clarify which axiom you see duplicated?